src/vivarium/framework/lookup.py

```diff
@@ -257,8 +263,11 @@ def build_table(self, data, key_columns, parameter_columns, value_columns) -> Ca
         return table
 
     def _build_table(self, data, key_columns, parameter_columns, value_columns):
-        return LookupTable(data, self._pop_view_builder, key_columns, parameter_columns,
-                           value_columns, self._interpolation_order, self.clock, self._extrapolate)
+        table_number = len(self.tables)
```

Comment:
New.  Giving generic names to tables so I can do something reasonable with them when constructing the dependency graph.